tox.ini

```
@@ -15,7 +15,7 @@ commands = flake8 tests/ potodo/
 
 [testenv:black]
 deps = black
-commands = black --check --diff tests/ potodo/
+commands = black --check --diff tests/ potodo/ --line-length 88
```

```suggestion
commands = black --check --diff tests/ potodo/
```
Not needed, 88 is the default.